terraform/modules/tf_lookup_tables_s3/main.tf

```diff
@@ -1,28 +1,45 @@
 data "aws_iam_policy_document" "streamalert_read_items_from_lookup_tables_s3" {
   statement {
-    actions   = ["s3:List*"]
-    resources = ["${local.s3_bucket_arns}"]
+    actions = ["s3:List*"]
+    # TF-UPGRADE-TODO: In Terraform v0.10 and earlier, it was sometimes necessary to
+    # force an interpolation expression to be interpreted as a list by wrapping it
+    # in an extra set of list brackets. That form was supported for compatibility in
+    # v0.11, but is no longer supported in Terraform v0.12.
+    #
+    # If the expression in the following list itself returns a list, remove the
+    # brackets to avoid interpretation as a list of lists. If the expression
+    # returns a single list item then leave it as-is and remove this TODO comment.
+    resources = [local.s3_bucket_arns]
```

Comment:
I think we can just use `local.s3_bucket_arns` here now (removing the `[]` around it)? same as below